pkg/ghclients/ghclients.go

```diff
@@ -80,13 +83,24 @@ func (g *GHClients) Get(i int64) (*github.Client, error) {
 	}
 	ctr := &httpcache.Transport{
 		Transport:           tr,
-		Cache:               httpcache.NewMemoryCache(),
+		Cache:               g.cache,
```

Comment:
Maybe I'm misunderstanding the usage here, but it somewhat worries me that we are sharing a memory cache across installation IDs. High possibility of OOM.

Reply:
Switching from multiple maps to a single map here. Still using the same amount of memory as before. I'm worried out OOM to, which is why I am trying to log the size.